primehub/admin_groups.py

```diff
@@ -120,6 +127,10 @@ def validate_resource_type(data: dict, user_field: str, project_field: str, acce
             raise PrimeHubException(
                 f'{user_field} should be less than or equal to {project_field}')
 
+    # fill default value
+    if user_quota is None and group_quota is None:
```

Comment:
Please don't mix the validation and auto-filling feature 

The auto-filling feature must be done before `validation`, so we could validate the inputs even someone users or our auto-filling feature put wrong values.